crates/astria-proto/src/lib.rs

```diff
@@ -1,3 +1,8 @@
+#[cfg(not(target_pointer_width = "64"))]
```

Comment:
I did not want to worry about casting usize to u64 and back. Astria Merkle is written entirely using `usize` so it works on any architecture. However, moving in between architectures is more tricky. So 64 bit it is.